Tests/UnitTests/Examples/EventData/MeasurementTests.cpp

```diff
@@ -65,184 +61,25 @@ BOOST_DATA_TEST_CASE(FixedBoundOne, bd::make(boundIndices), index) {
       BOOST_CHECK(!meas.contains(i));
     }
   }
-  BOOST_CHECK_EQUAL(meas.parameters(), params);
-  BOOST_CHECK_EQUAL(meas.covariance(), cov);
+  BOOST_CHECK_EQUAL(meas.effectiveParameters(), params);
+  BOOST_CHECK_EQUAL(meas.effectiveCovariance(), cov);
   BOOST_CHECK_EQUAL(meas.sourceLink().template get<TestSourceLink>(),
                     sourceOrig);
 }
 
-BOOST_AUTO_TEST_CASE(FixedBoundAll) {
+BOOST_AUTO_TEST_CASE(VariableBoundAll) {
   auto [params, cov] = generateBoundParametersCovariance(rng);
-  auto meas = makeFixedSizeMeasurement(source, params, cov, eBoundLoc0,
-                                       eBoundLoc1, eBoundPhi, eBoundTheta,
-                                       eBoundQOverP, eBoundTime);
+  auto meas = makeVariableSizeMeasurement(source, params, cov, eBoundLoc0,
+                                          eBoundLoc1, eBoundPhi, eBoundTheta,
+                                          eBoundQOverP, eBoundTime);
 
   BOOST_CHECK_EQUAL(meas.size(), eBoundSize);
   for (auto i : boundIndices) {
     BOOST_CHECK(meas.contains(i));
   }
-  BOOST_CHECK_EQUAL(meas.parameters(), params);
-  BOOST_CHECK_EQUAL(meas.covariance(), cov);
-  BOOST_CHECK_EQUAL(meas.sourceLink().get<TestSourceLink>(), sourceOrig);
-}
-
-namespace {
-// example data for phi residual tests. each entry contains
-//
-//     measured, reference, expected residual
-//
-const std::vector<std::tuple<double, double, double>> kPhiDataset = {
-    // measurement and reference in bounds and close
-    {0.5, 0.75, -0.25},
-    // measurement and reference in bounds but at different edges
-    {0.25, 2 * M_PI - 0.25, 0.5},
-    {2 * M_PI - 0.125, 0.125, -0.25},
-    // measurement in bounds, reference ouf-of-bounds, both near lower edge
-    {0.25, -0.25, 0.5},
-    // measurement in bounds, reference ouf-of-bounds, both near upper edge
-    {2 * M_PI - 0.25, 2 * M_PI + 0.25, -0.5},
-    // measurement out-of-bounds, reference in bounds, both near lower edge
-    {-0.25, 0.25, -0.5},
-    // measurement out-of-bounds, reference in bounds, both near upper edge
-    {2 * M_PI + 0.25, 2 * M_PI - 0.25, 0.5},
-};
-}  // namespace
-
-BOOST_DATA_TEST_CASE(BoundResidualsPhi, bd::make(kPhiDataset), phiMea, phiRef,
```

Comment:
Do we need a replacement for this test or is this covered indirectly elsewhere?

Reply:
I have killed the `residuals` method so these checks are gone. I will check again if the others can be recycled

Reply:
The others are free measurements which I also killed. I will recover the `VariantBound` one